src/extensions/default/CodeFolding/foldhelpers/foldgutter.js

```diff
@@ -310,7 +312,15 @@ define(function (require, exports, module) {
         });
     }
 
+    function remove(cm) {
+        var gutters = cm.getOption("gutters").slice(0);
+        var index = gutters.indexOf("CodeMirror-foldgutter");
+        gutters.splice(index, 1);
+        cm.setOption("gutters",  gutters);
+        CodeMirror.defineOption("foldGutter", false, null);
+    }
```

Comment:
Since this is basically the reverse of `createGutter()` in main.js, it seems like the two functions should live in the same file.  Could you either move that one here or move this there?